mobile/native/src/event/api.rs

```diff
@@ -190,3 +195,25 @@ pub struct Balances {
     pub on_chain: u64,
     pub off_chain: Option<u64>,
 }
+
+#[frb]
+#[derive(Clone)]
+pub enum FundingChannelTask {
+    Pending,
+    Funded,
+    Failed(String),
+    OrderCreated(String),
```

Comment:
That's a different task that you can subscribe to. Async order pending.

The rest is equal to the task status states.